app/controllers/entities_controller.rb

```diff
@@ -0,0 +1,78 @@
+class EntitiesController < ApplicationController
+  before_action :set_entity, only: %i[show edit update destroy]
+
+  # GET /entities or /entities.json
+  def index
+    @group = current_user.groups.includes(:entities).find(params[:group_id])
+  end
+
+  # GET /entities/1 or /entities/1.json
+  def show; end
+
+  # GET /entities/new
+  def new
+    @group = current_user.groups.find(params[:group_id])
+    @entity = Entity.new
+  end
+
+  # GET /entities/1/edit
+  def edit; end
+
+  # POST /entities or /entities.json
+  def create
+    @entity = Entity.new(entity_params.except(:group_ids, :group_id))
+
+    entity_params[:group_ids].each do |group_id|
+      @entity.groups << current_user.groups.select { |group| group.id == group_id.to_i }
+    end
+
```

Comment:
- [ ] Based on the project requirments, when  a user click (or taps) the "Save" button to create a new transaction(entity), It should re-direct  to the transactions page for that category. In your case, it leads back to category page. Kindly fix that  on the view 👍 

### Initial page in creation:
![image](https://github.com/IsmaelMastronardi/budget-app/assets/50990925/b4e0a081-ad4f-4649-a6ca-cfff810b2dbd)

### Expected next navigation on save:
![image](https://github.com/IsmaelMastronardi/budget-app/assets/50990925/467f13e4-f755-42c8-a18f-8db6ed44ee37)

### Page it leads to instead:
![image](https://github.com/IsmaelMastronardi/budget-app/assets/50990925/c874502e-405e-4d0a-9207-d6ff47c24ede)

## Requirements:
![image](https://github.com/IsmaelMastronardi/budget-app/assets/50990925/7ce4f766-5a1f-4069-8414-bbd0dd2ecf4f)